.gitignore

```diff
@@ -12,7 +12,3 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
-
-# PDCLI binary
-pdcli
```

Comment:
We should keep pdcli binary in .gitignore, we don't want to check in binary.

Reply:
@supreeth7  Did you do it intentionally? I think you should have run rebase while pulling changes from upstream.

Reply:
Yes Pooja, it was intentional. By adding 'pdcli' to .gitignore it stops tracking the entire cmd/pdcli folder. So I've tweaked the make file commands to create the binary directly in the ${GOPATH}/bin itself making it easier to natively run the pdcli command. For tests, the binary is created in the project root and deleted automatically upon completion.

Reply:
@supreeth7  try `./pdcli`

Reply:
Nope, that doesn't work. :)